index.js

```diff
@@ -36,10 +36,17 @@ function isMatch(pattern, options) {
     };
   }
 
-  if (Array.isArray(pattern) || isObject(pattern)) {
+  if (Array.isArray(pattern)) {
+    return function (val) {
+      return mm(val, pattern, options).length !== 0;
```

Comment:
`.isMatch` accept only string values and string pattern. I use micromatch() to allow to pass multiple values for single pattern

for array pattern I again use `mictomatch()` because same reason
we cant use `.any()` for array pattern, it have different meaning